DOCS/man/options.rst

```diff
@@ -2755,9 +2755,12 @@ Subtitles
     of subtitles across seeks, so after a seek libass can't eliminate subtitle
     packets with the same ReadOrder as earlier packets.
 
-``--teletext-page=<1-999>``
+``--teletext-page=<-1-999>``
     This works for ``dvb_teletext`` subtitle streams, and if FFmpeg has been
```

Comment:
while you're editing this could you add a real description? It doesn't actually say what the option does.

Reply:
Something like the commit I just pushed?